.byebug_history

```diff
@@ -0,0 +1,17 @@
+exit
+delay(queue)
+queue
+exit
+delay
+exit
+delay
+exit
+delay
+exit
+exitr
+self.delay
+self
+options
+hash
+@queues
+name
```

Comment:
@lucas-ibotta it is probably better to `~/.gitignore` this file. I believe it is more env specific. WDYT?